sklearn/ensemble/tests/test_bagging.py

```diff
@@ -723,3 +723,37 @@ def test_max_samples_consistency():
                                 max_features=0.5, random_state=1)
     bagging.fit(X, y)
     assert_equal(bagging._max_samples, max_samples)
+
+
+def test_set_oob_score():
+    # Make sure the oob_score doesn't change when the labels change
+    # See: https://github.com/scikit-learn/scikit-learn/issues/8933
+    N = 50
```

Comment:
As a non-regression test, I don't see why this is any better than asserting that

```python
BaggingClassifier(oob_score=True).fit([[0], [1]] * 5, ['A', 'B'] * 5).oob_score_
```

is approximately 1... or, at a minimum, not 0, which the current bug gives.